### PR TITLE
Add serdes package in release note

### PR DESCRIPTION
**Pull request by MohamedSabthar**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 1]
```diff
@@ -86,6 +86,11 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 ##### `graphql` package
 
+##### `serdes` package
+
+- Introduced the `serdes` standard library package for serializing and deserializing Ballerina `anydata` subtypes.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:91]:
```suggestion
- Introduced the `serdes` standard library package for serializing and deserializing Ballerina `anydata` subtypes
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 2]
```diff
@@ -86,6 +86,11 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
...
+- Proto3 is the underlying technology used by this package to achieve serialization and deserialization.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:92]:
```suggestion
- Proto3 is the underlying technology used by this package to achieve serialization and deserialization
```

Comment (MohamedSabthar) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md]: Suggestion committed

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 1 above)
Comment (MohamedSabthar) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md]: Suggestion committed

**Review by praneesha — APPROVED** [review]

**Pull request merged**